src/odemis/driver/lle.py

```diff
@@ -370,7 +371,7 @@ def _getIntensityGY(self, intensities):
             green_i = None
 
         # Yellow has precedence over green
-        if yellow_i is not None and intensities[yellow_i] > 0:
+        if yellow_i is not None and intensities[yellow_i]: # don't use if None or 0
```

Comment:
How is it here checking that it is not zero?

Reply:
`intensities[yellow_i]` is considered `False` if it's `0`, or `None`.